.github/workflows/create-release.yml

```diff
@@ -0,0 +1,38 @@
+name: Create Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: read
+    if: contains(github.event.head_commit.message, 'Cut release v')
+    steps:
+      - uses: actions/github-script@v7
+        name: Read Cut release PR info and create release
+        with:
+          script: |
+            const { owner, repo, sha } = context.repo
+            const pulls = await github.rest.repos.listPullRequestsAssociatedWithCommit({
+              owner,
+              repo,
+              commit_sha: sha,
+            })
+            const { title, body } = pulls.data[0]
+            const version = title.split('Cut release ')[1]
+
+            await github.rest.repos.createRelease({
+              owner,
+              repo,
+              body,
+              tag_name: version,
+              name: version,
+              draft: true,
```

Comment:
Looks like if I don't set it to draft (and the release is published), the CI doesn't get triggered. I'm thinking this is some kind of prevention for bots triggering bots. What this means is we still have to go it publish. But it's at least a start!